shared/entity/type_test.go

```diff
@@ -48,7 +48,7 @@ func TestURL(t *testing.T) {
 			name:               "projects",
 			rawURL:             "/1.0/projects/my-project",
 			expectedEntityType: TypeProject,
-			expectedProject:    "",
+			expectedProject:    "my-project",
```

Comment:
Is there a test for the change in the previous commit?

Reply:
Hmmm I'll double check. This tests that the `ParseURL` and `(Type).URL` are each others inverses but I don't remember if it explicitly checks query parameters.

Reply:
I've updated the tests to have an "expected normalised URL". So rather than conditionally checking that the result of `(Type).URL` is the inverse of `ParseURL`, we check that the parsing works, and that creating a new URL from the result of parsing the original URL gives what we expect.